dll/win32/kernel32/client/sysinfo.c

```diff
@@ -557,6 +729,36 @@ GetSystemFirmwareTable(IN DWORD FirmwareTableProviderSignature,
                                    SystemFirmwareTable_Get);
 }
 
+#if (_WIN32_WINNT >= _WIN32_WINNT_WIN8)
+_Success_(return != FALSE)
+BOOL
+WINAPI
+GetFirmwareType(
+    _Out_ PFIRMWARE_TYPE FirmwareType)
+{
+  NTSTATUS Status;
```

Comment:
Indentation in this function is broken, please fix.

Reply:
Fixed.